machine_learning/benchmarks/base_classifier_benchmark.py

```diff
@@ -33,28 +38,13 @@ def __init__(self):
             "qasm_simulator": quantum_instance_qasm,
         }
 
-        self.dataset_1 = np.array(
-            [
-                [0.63332707, 0.05700334],
-                [-0.04218316, -0.74066734],
-                [-0.54731074, 0.6997243],
-                [-0.03254765, 0.68657814],
-                [0.57025591, 0.67333428],
-                [0.32978679, 0.90721741],
-                [0.28112104, -0.52329682],
-                [0.03209235, 0.05112333],
-                [0.46215367, 0.97636782],
-                [0.93945321, 0.09375981],
-                [-0.6546925, 0.1612654],
-                [0.38871208, 0.73535322],
-                [-0.72805702, 0.73124097],
-                [-0.79972062, -0.84444756],
-                [0.87636701, -0.66912929],
-                [-0.08563266, 0.79913683],
-                [0.31805884, -0.84938654],
-                [0.96364301, 0.86688318],
-                [-0.50482284, -0.64370197],
-                [-0.41502205, 0.38414452],
-            ]
-        )
-        self.datasets = {"dataset_1": self.dataset_1}
+        self.datasets = {
+            "dataset_synthetic_classification": {
```

Comment:
This is a classification specific base class `_classification` is not required, we already know we'll classify.

Reply:
Changed all instances of `dataset_synthetic_classification` to `dataset_synthetic`